moveit2.repos

```diff
@@ -2,7 +2,7 @@ repositories:
   geometric_shapes:
     type: git
     url: https://github.com/ros-planning/geometric_shapes
-    version: 2.1.2
+    version: ros2
```

Comment:
We should make a new release for geometric shapes to fix buildfarm errors and use the tag again.